docs/rules/prefer-math-trunc.md

```diff
@@ -1,23 +1,46 @@
-# Enforce the use of `Math.trunc` instead of `| 0`
+# Enforce the use of `Math.trunc` instead of bitwise operators
 
-Enforces a convention of using [`Math.trunc()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc) instead of `| 0` (bitwise OR with 0) for clarity and more reliable results.
+Enforces a convention of using [`Math.trunc()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc) instead of `| 0` ([`bitwise OR`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR) with 0) or `~~x` (two [`bitwise NOT`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_NOT)) for clarity and more reliable results.
 
-The `| 0` hack helps truncate numbers but is not clear and does not work in [some cases](https://stackoverflow.com/a/34706108/11687747).
+These hacks helps truncate numbers but they are not clear and do not work in [some cases](https://stackoverflow.com/a/34706108/11687747).
```

Comment:
```suggestion
These hacks help truncate numbers, but they are not clear and do not work in [some cases](https://stackoverflow.com/a/34706108/11687747).
```

Reply:
Ups sorry I did not see I could commit your suggestion directly 😄 well it is now pushed. Thanks!